ESCPOS_NET/Emitters/BaseCommandEmitter/ImageCommands.cs

```diff
@@ -1,11 +1,8 @@
-﻿using SixLabors.ImageSharp;
-using SixLabors.ImageSharp.PixelFormats;
-using SixLabors.ImageSharp.Processing;
-using SixLabors.ImageSharp.Advanced;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 using ESCPOS_NET.Utilities;
 using System;
+using System.Drawing;
```

Comment:
What's the performance like of system.drawing vs the imagesharp lib?

Reply:
Em um teste simples que fiz com Stopwatch. O system.drawing rodou em 957ms. O imagesharp lib rodou em 1530ms. Mas só fiz esse teste rápido para dar uma resposta aqui.

Reply:
In a simple test I did with Stopwatch. System.drawing ran at 957ms. The imagesharp lib ran at 1530ms. But I just did this quick test to give an answer here.